studygroups/tasks.py

```diff
@@ -215,7 +215,9 @@ def send_final_learning_circle_report(study_group):
         report_path = reverse('studygroups_final_report', kwargs={'study_group_id': study_group.id})
         report_url = domain + report_path
         recipients = study_group.application_set.values_list('email', flat=True)
-        to = list(recipients)
+        organizers = get_study_group_organizers(study_group)
+        organizers_emails = [organizer.email for organizer in organizers]
+        to = list(recipients) + organizers_emails
```

Comment:
@s-kennedy Does this email send to all signups with their email in the `to` field? For emails we sent to learners, we shouldn't disclose email addresses of other learners.

Reply:
good point, I'll fix that.

Reply:
Thanks!